frontend/src/lib/components/game/tooltips/PopulationTooltip.svelte

```diff
@@ -24,34 +24,38 @@
 				Your population on <span class="font-semibold">{planet.name}</span> is
 				<span class="font-semibold">{planet.spec.population.toLocaleString()}</span> ({(
 					planet.spec.populationDensity * 100
-				).toFixed()}% of capacity).
+				).toFixed()}% of capacity).<br />
```

Comment:
instead of using <br>, please wrap anything that needs to be a separate paragraph in a <p>content</p>

Reply:
Ok I'll get on to that once I get back home and set up the local instance again.